source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -489,7 +489,7 @@ function theme_scripts_styles() {
 		/* translators: Subsets can be any of cyrillic, cyrillic-ext, greek, greek-ext, vietnamese, latin, latin-ext. */
 		$subsets = _x( 'Latin', 'Heading font subsets, comma separated', 'wporg' );
 		// All headings.
-		global_fonts_preload( 'IBM Plex Sans, IBM Plex Sans SemiBold, Inter', $subsets );
+		global_fonts_preload( 'Inter', $subsets );
```

Comment:
Replace with Garamond, we preload that in [main](https://github.com/WordPress/wporg-main-2022/blob/2a019558aa69f5838a2d862378ccdfd52f66faae/source/wp-content/themes/wporg-main-2022/functions.php#L96)?

Reply:
Good idea, added in https://github.com/WordPress/wporg-developer/pull/493/commits/88b5d42761098676fd03d0a61f25f87cda6f214e